### PR TITLE
KOGITO-575 - Index Job data

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

File: data-index/data-index-service/src/main/resources/application.properties [hunk 1]
```diff
@@ -34,6 +34,11 @@ mp.messaging.incoming.kogito-usertaskdomain-events.connector=smallrye-kafka
 mp.messaging.incoming.kogito-usertaskdomain-events.topic=kogito-usertaskinstances-events
 mp.messaging.incoming.kogito-usertaskdomain-events.value.deserializer=org.kie.kogito.index.messaging.KogitoUserTaskCloudEventDeserializer
 mp.messaging.incoming.kogito-usertaskdomain-events.bootstrap.servers=localhost:9092
+
+mp.messaging.incoming.kogito-jobs-events.connector=smallrye-kafka
```
Comment (cristianonicolai) [inline data-index/data-index-service/src/main/resources/application.properties]: @ricardozanini heads up for a new Kafka endpoint in the Data Index

**Comment by mswiderski** [comment]
@cristianonicolai @tiagodolphine overall it looks nice

the only thing that is not in sync is the topic name, in jobs-service it is `kogito-job-service-job-status-events-topic` and in data index it is `kogito-jobs-events`

so this needs to be aligned and I am in favour of `kogito-jobs-events`, thoughts?

if we agree I'd say we should make the change in this PR.

**Comment by cristianonicolai** [comment]
@mswiderski @tiagodolphine +1 to use `kogito-jobs-events`

**Comment by tiagodolphine** [comment]
@mswiderski @cristianonicolai I'm ok to change the topic name +1

**Comment by mswiderski** [comment]
@cristianonicolai @tiagodolphine I updated the topic name and will wait for CI to finish build and then merge it

File: data-index/data-index-service/src/main/resources/application.properties (hunk 1 above)
Comment (ricardozanini) [reply data-index/data-index-service/src/main/resources/application.properties]: Thanks! Opened a follow up JIRA: https://issues.redhat.com/browse/KOGITO-904

**Pull request merged**